src/app/account/account.types.ts

```diff
@@ -1,3 +0,0 @@
-import { User } from '@/app/admin/users/users.types';
-
-export type Account = User;
```

Comment:
We want to keep this empty file ?

Reply:
I don't remember why I removed this 😅 
I will either remove the file or revert the change

Reply:
Oh it just that the file was moved from `app` to `spa`